backoffice/backoffice/workflows/api/views.py

```diff
@@ -100,6 +101,38 @@ def create(self, request, *args, **kwargs):
             )
 
 
+class DecisionViewSet(viewsets.ModelViewSet):
+    queryset = Decision.objects.all()
+    serializer_class = DecisionSerializer
+
+    def get_queryset(self):
+        status = self.request.query_params.get("status")
+        if status:
+            return self.queryset.filter(status__status=status)
+        return self.queryset
+
+    def create(self, request, *args, **kwargs):
+        workflow_id = request.data.get("workflow_id")
+        action = request.data.get("action")
+
+        if not all([workflow_id, action]):
```

Comment:
That's why we have the serializer :) we can check if it's valid and then save

Reply:
Reviewing. Same approach was also implemented in the `WorkflowTicketViewSet` should be reviewed as well
https://github.com/inspirehep/backoffice/blob/main/backoffice/backoffice/workflows/api/views.py#L84
Ticket created https://github.com/cern-sis/issues-inspire/issues/544

Reply:
👍 thanks for creating the issue, on django it's a best practise to use the serializer for validation